postgrest.cabal

```diff
@@ -148,9 +153,6 @@ library
   if !os(windows)
     build-depends:
       unix
-      , directory >= 1.2.6 && < 1.4
-    exposed-modules:
-      PostgREST.Unix
```

Comment:
@develop7 Any concerns on windows? Maybe we should allow running tests on windows to be extra sure (other PR).

Reply:
That'd be only fair, since we do release windows binaries

Reply:
We cannot enable all tests on Windows right off the bat, but specs & doctests should be fine